scripts/compare_versioned_types.py

```diff
@@ -10,10 +10,27 @@
 
 from compare_versioned_items import run_comparison
 
+error_msg = "Please see this issue https://github.com/MinaProtocol/mina/issues/11780 for pointers on how to fix this error. It might be also helpful to look in this file: `scripts/compare_verioned_types.py"
+
+# This is the list of file while has never been changed before
+# and they haven't run through the CI before, and then fail 
+# when modified. To add new files, just add them to this list.
+skip_list = ["src/nonconsensus/snark_params/tick.ml"]
+
+def skip(original, modified):
```

Comment:
Quoting the above comment here:
> This is the list of file while has never been changed before
> and they haven't run through the CI before, and then fail when modified.

Some observations:
1. The `modified` parameter isn't being used in the `skip` function.
2. How are you accounting for failures when the skip list is modified?
3. Can you add some documentation to the `skip` function? It's not clear to me what both parameters mean.

Reply:
The `skip` function determines if a particular file should be skipped  version compatibility check in the CI check.
`modified` is the name of the file we have modified in the branch of a PR.
`original` is the name of the equivalent file in at the HEAD of the branch you want to merge into (this is usually develop or compatible).
They basically have the same names, but different paths. I'll add these as documentation to the file.

> 2. How are you accounting for failures when the skip list is modified?

What do you mean by this?

Reply:
Do you need `modified` variable at all? I don't see it being used anywhere inside the function.